network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -0,0 +1,25 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+	<div class="tw-row tw-py-4">
+		{% for block in self.cards %}
+          {% with card=block count=self.cards|length %}
```

Comment:
**Question** Can you add some context to what the count is used for?

Reply:
I've removed this as we aren't using it anymore, good catch!